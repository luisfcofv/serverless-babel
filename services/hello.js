export const hello = ({ name }) => `Hello ${name}! Your function executed successfully`;
export default { hello };
