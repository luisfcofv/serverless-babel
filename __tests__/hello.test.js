import { hello } from '../services/hello';

test('Hello World', () => {
  const service = hello({ name: 'World' });
  expect(service).toMatchSnapshot();
});
