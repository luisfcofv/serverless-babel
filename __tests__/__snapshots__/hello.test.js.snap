// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World 1`] = `"Hello World! Your function executed successfully"`;
